homeassistant/components/tplink/README.md

```diff
@@ -0,0 +1,31 @@
+# TPLink Integration
+
+This document covers details that new contributors may find helpful when getting started.
+
+## Modules vs Features
+
+The kasa library which this integration depends on exposes functionality via modules and features.
+The `Module` apis encapsulate groups of functionality provided by a device,
+e.g. Light which has multiple attributes and methods such as `set_has`, `brightness` etc.
```

Comment:
`set_has`?

Reply:
Now fixed in https://github.com/home-assistant/core/pull/120025